libraries/AC_Fence/AC_Fence.h

```diff
@@ -166,4 +177,9 @@ class AC_Fence
     bool            _boundary_create_attempted = false; // true if we have attempted to create the boundary array
     bool            _boundary_loaded = false;       // true if boundary array has been loaded from eeprom
     bool            _boundary_valid = false;        // true if boundary forms a closed polygon
+
+    // beacon fence variables
+    Vector2f        *_beacon_boundary;              // array of boundary points.  Note: point 0 is the return point
```

Comment:
If it is not intentional, I think that it is better to initialize like _boundary related variables above.